Marlin/planner.h

```diff
@@ -395,10 +395,16 @@ class Planner {
 
     #if ENABLED(ULTRA_LCD)
 
-      static millis_t block_buffer_runtime() {
+      static uint16_t block_buffer_runtime() {
         CRITICAL_SECTION_START
           millis_t bbru = block_buffer_runtime_us;
         CRITICAL_SECTION_END
+        // To translate µs to ms a division by 1000 would be required.
+        // We introduce 2.4% error her by dividing by 1024.
+        // Does not matter because block_buffer_runtime_us is already an, too small, estimation.
```

Comment:
If the estimation is on the low side, is it possible to err on the side of _too high_ instead of making it more low?